liquidity/liquidity.go

```diff
@@ -78,6 +86,39 @@ func (p Parameters) validate() error {
 	return nil
 }
 
+// ExistingSwap provides information about a swap that has been dispatched.
+type ExistingSwap struct {
+	// SwapHash is the hash used for the swap.
+	SwapHash lntypes.Hash
+
+	// State is the current state of the swap.
+	State loopdb.SwapState
+
+	// Type indicates the type of swap.
+	Type swap.Type
+
+	// Channels is the set of channels that a loop out swap is using.
+	Channels []lnwire.ShortChannelID
+
+	// Peer is the last hop set for loop in (if any).
+	Peer *route.Vertex
+}
```

Comment:
I would be careful mixing loop in and out again here, as we've been separating data structures elsewhere